### ISSUE TITLE
Content Editable elements in HTML emails are actually editable

### ISSUE THREAD

**Issue by markbao** [nylas/N1#1919]
![contenteditable](https://cloud.githubusercontent.com/assets/27594/14406427/1456876c-fe75-11e5-98bd-3832c10a0d3d.gif)

Content Editable elements in HTML emails are actually editable in the message view. Outlook iOS has a bug where if you write a reply and hit `> Load full message`, it will embed your reply text inside a contenteditable div. Yeah, it's stupid. But Gmail strips this out, whereas N1 keeps it.

Reproduce contenteditable with Outlook iOS:

1. Find an email.
2. Reply to it.
3. Tap `> Load full message`.
4. Type some stuff.
5. Send message.

Reproduce editability in N1:

1. Find the sent email.
2. Notice that you can click inside your reply. It's a text box and you can type in there. (Type carefully, since whatever you type is interpreted by N1 as keyboard commands.)

Gmail makes this contenteditable non-editable. It just shows as regular text. Seems like N1 should be doing the same.

##### Are there any related issues?

No

##### What operating system are you using?
10.11.4 (15E65)

##### What version of N1 are you using?
Version 0.4.19-d41e72c (0.4.19-d41e72c)

--

**Bug?**
##### Do you have any third-party plugins installed?
No

##### Is the issue related to a specific email provider (Gmail, Exchange, etc.)?
No

##### Is the issue reproducible with a particular attachment, message, signature, etc?

See above

**Comment by rittbys** [comment]
Can you attatch an example email that is editable?

**Comment by markbao** [comment]
Sent in the message to N1 support, and also attaching HTML here

```
Return-Path: <mark@decisionsciences.[snip]>
Received: from mail.outlook.com [snip]
        by smtp.gmail.com with ESMTPSA id [snip]
        for <mark@markbao.[snip]>
        (version=[snip]);
        Sat, 09 Apr 2016 14:01:38 -0700 (PDT)
Date: Sat, 9 Apr 2016 21:01:37 +0000 (UTC)
From: Mark Bao <mark@decisionsciences.[snip]>
To: Mark Bao <mark@markbao.[snip]>
Message-ID: <[snip]@mail.outlook.com>
In-Reply-To: <[snip]@mail.outlook.com>
References: <[snip]@mail.outlook.com>
Subject: Re: Testing
MIME-Version: 1.0
Content-Type: multipart/alternative; 
	boundary="----=_Part_7080_326667712.1460235697887"
X-Mailer: Outlook for iOS and Android

------=_Part_7080_326667712.1460235697887
Content-Type: text/plain; charset=us-ascii
Content-Transfer-Encoding: 7bit

Hey. This is editable dude



    _____________________________
From: Mark Bao <mark@decisionsciences.[snip]>
Sent: Saturday, April 9, 2016 5:01 PM
Subject: Testing
To: Mark Bao <mark@markbao.[snip]>








  
------=_Part_7080_326667712.1460235697887
Content-Type: text/html; charset=utf-8
Content-Transfer-Encoding: 7bit


    <div id="compose" contenteditable="true" style="padding-left: 20px; padding-right: 20px; padding-bottom: 8px;"><div>Hey. This is editable dude<br><br><div class="acompli_signature"></div><br></div></div>
    <div class="gmail_quote">_____________________________<br>From: Mark Bao &lt;<a dir="ltr" href="mailto:mark@decisionsciences.[snip]" x-apple-data-detectors="true" x-apple-data-detectors-type="link" x-apple-data-detectors-result="1">mark@decisionsciences.[snip]</a>&gt;<br>Sent: Saturday, April 9, 2016 5:01 PM<br>Subject: Testing<br>To: Mark Bao &lt;<a dir="ltr" href="mailto:mark@markbao.[snip]" x-apple-data-detectors="true" x-apple-data-detectors-type="link" x-apple-data-detectors-result="4">mark@markbao.[snip]</a>&gt;<br><br><br><div><br><br><div class="acompli_signature"></div><br></div><br><br></div>
  
------=_Part_7080_326667712.1460235697887--
```

**Comment by markbao** [comment]
Issue persists in Version 0.4.32-1de17e7 (0.4.32-1de17e7)

**Comment by emorikawa** [comment]
Confirmed this still exists in 0.4.45. @GM-Polyakov showing another report of this. Working on fix. Thanks @markbao for extra debug info.

**Comment by bengotow** [comment]
Was able to reproduce this with an email sent from the latest version of Outlook for Mac. Very strange that they are leaving `contenteditable` attributes in the messages they send. Fixed in e40239f6b9ed9a1fc5154fd466e89f59b0331368 and pending release.

**Comment by jackiehluo** [comment]
Hey! We've pushed an update (`0.4.47`) that should fix this issue—let us know if you see it again! Thanks!
